### ISSUE TITLE
Go to definition duplicate view

### ISSUE THREAD

**Issue by husanjun** [sublimelsp/LSP#1526]
**Describe the bug**
In the same location , when I hit Go to the definition again , the view selection pops up , but they all have the same path except that the drive letter case is different

**To Reproduce**
1. click Go to definition
2. close file 
3. click Go to definition again

**Environment:**
- OS: win7
- Sublime Text version: 4094
- LSP version: 4070-1.2.3
- Language servers used: intelephense

**Comment by jfcherng** [comment]
Wondering will a special treatment for Windows be prefered. Like

```py
def unique_windows_paths(paths: List[str]) -> List[str]:
    seen = collections.OrderedDict()

    for path in paths:
        path = re.sub(r"[/\\]+", r"\\", path)  # unify OS seps
        path_lower = path.lower()
        if path_lower not in seen:
            seen[path_lower] = path

    return list(seen.values())
```

**Comment by rwols** [comment]
> path_unified = re.sub(r"[/\\]+", r"\\", path).lower()

Does that work when using WSL?

**Comment by jfcherng** [comment]
> > path_unified = re.sub(r"[/\]+", r"\", path).lower()
> 
> Does that work when using WSL?

Hmm, I didn't try. But `path_unified` is just for checking unique paths, not actually being used as returned values.

**Comment by ZLOY5** [comment]
Have same issue

OS: Windows 10
Sublime Text version: 3211
LSP version: 3154-0.14.3
Language servers used: https://github.com/sumneko/lua-language-server

**Comment by husanjun** [comment]
Solved in intelephense 1.7 
bmewburn/vscode-intelephense#1555
